src/webgpu/shader/validation/wgsl/duplicate-var-name-within-func.fail.wgsl

```diff
@@ -1,13 +1,11 @@
-# v-0014 - This fails because of the duplicated `a` variable.
+# v-0014 - This passes because variable `a` is redeclared.
```

Comment:
I know we agreed on lexical scoping, but did we agree to allow shadowing? @dj2

Reply:
As far as I remember, shadowing is _not_ allowed.

Reply:
sg, in that case we should test both the old test (siblings, passes) and new test (nested, fails)